src/app/util/ember-compatibility-functions.cpp

```diff
@@ -134,6 +136,15 @@ EmberAfAttributeType BaseType(EmberAfAttributeType type)
     }
 }
 
+Protocols::InteractionModel::Status CheckAccessControl(const SubjectDescriptor & aSubjectDescriptor, EndpointId aEndpoint, ClusterId aCluster, bool aWrite)
+{
+    RequestPath requestPath = { .cluster = aCluster, .endpoint = aEndpoint };
+    // TODO: get required privilege using ember APIs, as it could be custom
+    Privilege privilege = aWrite ? Privilege::kOperate : Privilege::kView;
```

Comment:
This is 'minimum' privilege right? If so, might want to rename it to indicate that.

Reply:
No it is the privilege required. They are a set. There is no minimum/maximum as they are not a linear list. Instead, multiple privileges are granted. So this is the privilege required.